sql/bcnm_treasure_chests.sql

```diff
@@ -421,3 +421,16 @@ INSERT INTO bcnm_treasure_chests VALUES (1152,3,17051654);
 INSERT INTO bcnm_treasure_chests VALUES (1153,1,17051660); -- shadows_of_the_mind
 INSERT INTO bcnm_treasure_chests VALUES (1153,2,17051666);
 INSERT INTO bcnm_treasure_chests VALUES (1153,3,17051672);
+
+-- ----------------------------
+-- Temenos
+-- ----------------------------
+INSERT INTO bcnm_treasure_chests VALUES (1298,3,16928785); -- Temenos - Western Tower
```

Comment:
I don't know if this changed with the BCNM rewrite, but I think the reason these were never there was because of the BCNM system counting the battlefield as "over with" on chest open when limbus lets you continue to roam around after until the time runs out. I f this isn't a problem, ignore+"resolve" this comment :+1:

Reply:
I was getting missing treasure chest errors without them. I also pulled the core stuff out of the bcnm loot rolling into the Armoury Crate file to get it to actually make the loot. Without doing that it would only give loot on the last chest on 7th floor. Not sure why, since it's doing the exact same thing. It does currently warp you out to the lobby after you open the last chest.

Reply:
Someone more familiar with how this should work is going to have to review then. I'm not confident I would guide you to the correct solution.

Reply:
I checked out some recorded streams and it looks like retail does warp you out when you open the last chest.